src/SFA.DAS.AssessorService.Settings/WebConfiguration.cs

```diff
@@ -22,6 +22,8 @@ public class WebConfiguration : IWebConfiguration
 
         [JsonRequired] public string IfaApiClientBaseUrl { get; set; }
 
+        [JsonRequired] public string ProviderEventsClientBaseUrl { get; set; }
```

Comment:
Are you aware about the das-employer-config project?

I can walk you through it, however you'll need to update both the schema file and the json file in there then get one of us plus a person from DevOps to approve